src/main/java/hello/helper/GetResponse.java

```diff
@@ -0,0 +1,33 @@
+package hello.helper;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class GetResponse {
+    @Getter @Setter private Integer fixId;
+    @Setter private List<GetResponseIssue> issues;
+    @Getter @Setter private String s3Link;
+    @Getter @Setter private String status;
```

Comment:
Never set Http status code inside the body of the response.
While consuming the response from a request we first check response.status
and depending on the status code, the body of the response gets processed.

Reply:
this response is built for user not getting from codegen, this will directly be converted to JSON.